WordPressShared/WordPressShared/Core/Views/WPStyleGuide+DynamicType.swift

```diff
@@ -0,0 +1,182 @@
+/// Extension on WPStyleGuide to use Dynamic Type fonts.
+///
+extension WPStyleGuide {
+    static let defaultTableViewRowHeight: CGFloat = 44.0
+
+    static let notoLoaded = { () -> Bool in
+        WPFontManager.loadNotoFontFamily()
+        return true
+    }()
+
+    /// Configures a table to automatically resize its rows according to their content.
+    ///
+    /// - Parameters:
+    ///     - tableView: The tableView to configure.
+    ///
+    public class func configureAutomaticHeightRowsForTableView(_ tableView: UITableView) {
+        tableView.rowHeight = UITableViewAutomaticDimension
+        tableView.estimatedRowHeight = defaultTableViewRowHeight
+    }
+
+    /// Configures a label with the default system font with the specified style.
+    ///
+    /// - Parameters:
+    ///     - label: The label to configure.
+    ///     - style: The desired UIFontTextStyle.
+    ///
+    public class func configureLabel(_ label: UILabel, forTextStyle style: UIFontTextStyle) {
```

Comment:
`forTextStyle` still feels a bit Obj-C-ish to me. Thoughts on `configureLabel(_ label: textStyle style:)`?